test/cases/helper_cockroachdb.rb

```diff
@@ -1,5 +1,5 @@
-require 'bundler/setup'
-Bundler.require :development
+require 'bundler'
+Bundler.setup
```

Comment:
This is taken from https://github.com/rails/rails/blob/7f338b9082fe3488f681c171b93c371cfc951f22/load_paths.rb.

With the previous version we would load the `railties/lib/rails.rb` file from rails. This file defines `Rails.env`. And having this method defined keep a cached version of the env, failing tests that are trying to redefine the rails env...

Reply:
nice improvement